### DISCUSSION TITLE
Custom mount of location to a mountPoint

### DISCUSSION THREAD

**Discussion by madalinignisca** [lima-vm/lima#2435] [Q&A] [answered]
I'm very confused on mounts.

I'm trying to mount one specific directory to make it available under a specific directory.

It gets mounted as expected, but the default shell login changes behaviour: it logins to the same user path like my MacOS host.

VM: user in
`/home/madalin.linux` as I saw it gets created.
mount:
  - location: '/Volumes/Workspace/git'
    mountPoint: '/home/madalin.linux/git'
    writable: true

The above works.

But: when I login, it defaults in an non-existing directory `/Users/madalin`. Why is lima generating ssh config trying to login the user to some location and not it's default home?

**Comment by jandubois** [comment, marked as answer]
The `lima shell` command by default opens a shell in the same directory that you are in on the host. This is by design.

You can override this

```console
$ limactl shell --workdir /home/madalin.linux default
madalin@lima-default:~$
```

Or if you use the `lima` wrapper:

```console
$ export LIMA_WORKDIR=/home/madalin.linux
$ lima
madalin@lima-default:~$
```
